WORKSPACE

```diff
@@ -15,7 +15,7 @@ git_repository(
 
 git_repository(
     name = "entangled",
-    commit = "46cab95db5fa1e03274446d9f2df9a4b7ba12462",
+    commit = "d4dbc80a216dd7bac7bec7d2aafdad9142b8b3fa",
```

Comment:
Since the stable version is released. We should use it with release tag instead of commit hash:
```
git_repository(
    name = "entangled",
    remote = "https://github.com/iotaledger/entangled.git",
    tag = "cclient-v1.0.0-beta",
)
```